api/services/transfer/v1/transfer.proto

```diff
@@ -35,5 +35,6 @@ message TransferRequest {
 
 message TransferOptions {
 	string progress_stream = 1;
+	repeated string platforms = 2;
```

Comment:
This should use `types.Platform`, we aren't going to use the formatted strings in the API